test/util/validator.test.js

```diff
@@ -12,43 +12,59 @@ test('validateInputJson', () => {
   }
 
   const stringFields = [ 'currencyCode', 'partnerId', 'accessKey', 'secretKey' ]
-  const enumFields = [ 'environment', 'endpoint' ]
-  const badStrings = [ 35, [], '' ]
-  const goodEnums = {
+  
+  const enums = {
```

Comment:
Move to valid enums test

Reply:
The enum object is used by multiple tests.